### PR TITLE
[PATCH makeoffer] Get the offer price from myLastOffer, not Order

### PR DESCRIPTION
**Pull request by dleve123**
Fixes https://artsyproduct.atlassian.net/browse/PURCHASE-704

* Do less de-structuring
* Updates from `yarn relay`

---

Check it out:

screen shot 2018-11-30 at 5 10 11 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 949fb6ac8b48feae693f5e93a175b10ec657f8d2

https://deploy-preview-1622--artsy-reaction.netlify.com

**Comment by dleve123** [comment]
So someone is assigned to this PR, so dunno why Peril is complaining? @orta Any idea what's up with that?

**Comment by zephraph** [comment]
I think it just doesn't remove the message after it's initially added. Likely you created the PR before you assigned someone and it picked the message up from there.

File: src/Apps/Order/Components/OfferSummaryItem.tsx [hunk 1]
```diff
@@ -31,6 +31,11 @@ export const OfferSummaryItemFragmentContainer = createFragmentContainer(
     fragment OfferSummaryItem_order on Order {
       itemsTotal(precision: 2)
```
Comment (ds300) [inline src/Apps/Order/Components/OfferSummaryItem.tsx]: I think you can delete this line now.

Comment (dleve123) [inline src/Apps/Order/Components/OfferSummaryItem.tsx]: totally, thanks!

Comment (dleve123) [reply src/Apps/Order/Components/OfferSummaryItem.tsx]: totally, thanks!

**Comment by sweir27** [comment]
Nice! Welcome to reaction/relay 😄.

**Pull request merged**